src/eins-hwinfo.c

```diff
@@ -91,7 +97,11 @@
  * array, containing details of the big and LITTLE cores. In practice, the
  * current implementation only reports the currently-active cores.
  */
-#define CPU_MODELS_EVENT "4a75488a-0d9a-4c38-8556-148f500edaf0"
+
+#define CPUINFO_TYPE_STRING "(sqd)"
+
+#define COMPUTER_HWINFO_TYPE_STRING "(" RAMINFO_TYPE_STRING \
+  ROOTFS_SPACE_TYPE_STRING "@a" CPUINFO_TYPE_STRING ")"
```

Comment:
```suggestion
#define CPUINFO_TYPE_STRING "a(sqd)"

#define COMPUTER_HWINFO_TYPE_STRING "(" RAMINFO_TYPE_STRING \
  ROOTFS_SPACE_TYPE_STRING "@" CPUINFO_TYPE_STRING ")"
```

Reply:
I keep original `CPUINFO_TYPE_STRING` and add `CPUINFO_ARRAY_TYPE_STRING`.  `CPUINFO_TYPE_STRING` will be used to create new CPU information array in fixed version.